x/osmolbp/client/cli/parse.go

```diff
@@ -0,0 +1,53 @@
+package cli
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+
+	"github.com/spf13/pflag"
+)
+
+type CreateLBPInputs createLBPInputs
```

Comment:
why type rename?

Reply:
without renaming type if i directly use `createLBPInputs`
getting 
`runtime: goroutine stack exceeds 1000000000-byte limit`
I'm not sure why this is happening but with renaming & type conversion it's working